src/shared/components/iframe-runtime.tsx

```diff
@@ -101,13 +101,15 @@ export const IframeRuntime: React.FC<IProps> =
         phoneRef.current.disconnect();
       }
     };
-  }, [url, authoredState, report, id, navImageUrl, navImageAltText]);
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  },[]);
 
+  const iframeStyle = wrapper === "scaler" ? iframeStyling : {width: "100%", height: iframeHeight, border: "none"};
   return (
-    <div>
-      <iframe ref={iframeRef} src={url} width="100%" height={iframeHeight} frameBorder={0} />
+    <>
+      <iframe ref={iframeRef} src={url} style={iframeStyle} className={wrapper === "scaler" ? css.subInteractiveIframe : ""} />
       { hint &&
         <div className={css.hint}>{renderHTML(hint)}</div> }
```

Comment:
For the scaler/fullscreen interactive the hint should be set on the host instead of shown inline like this. 

So I think the best approach is to pass a `setHint` function as a property to IframeRuntime, and move this rendering of the hint up into the scaffolded question and carousel runtime components.

The scaler/fullscreen interactive can implement setHint by calling `setHint` from the interactive-api.